### PR TITLE
feat(api): background selection

### PR DESCRIPTION
**Pull request by kobryl**

### CODE REVIEW

**Comment by kobryl** [comment]
Needs #6 to be done before merging

**Comment by AlicjaLukaszewicz** [comment]
Torch version is incorrect if there is a need for running the project on PC without CUDA .  Command `pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu` installs correct version for running on CPU. Not sure if requirements need to be updated in this case.

**Comment by kobryl** [comment]
> Torch version is incorrect if there is a need for running the project on PC without CUDA . Command `pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu` installs correct version for running on CPU. Not sure if requirements need to be updated in this case.

One of the requirements to run this project is CUDA so this is intentional.

File: image_segmentation/main.py
Comment (michalkr52) [inline image_segmentation/main.py:1]: The SAM checkpoint path is a local path that you have to replace, that will be addressed later

Comment (kobryl) [reply image_segmentation/main.py:1]: It will be converted to MD file in #3

**Review by michalkr52 — APPROVED** [review]
LGTM

**Pull request merged**